From e13f4e02e11afa45d8bcb6ceb46bd50c3b6c9aae Mon Sep 17 00:00:00 2001
From: Michael Tremer <michael.tremer@ipfire.org>
Date: Sun, 22 Apr 2012 14:44:51 +0200
Subject: [PATCH 6/8] Actually remove installed packages from database.

When removing packages, the metadata was not removed from
the local package database.
---
 python/pakfire/repository/system.py |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/python/pakfire/repository/system.py b/python/pakfire/repository/system.py
index 5dcdd04..ce3e979 100644
--- a/python/pakfire/repository/system.py
+++ b/python/pakfire/repository/system.py
@@ -62,6 +62,7 @@ class RepositorySystem(base.RepositoryFactory):
 
 	def rem_package(self, pkg):
 		# Remove package from the database.
+		self.db.rem_package(pkg)
 		self.index.rem_package(pkg)
 
 	@property
-- 
1.7.7.6

